frontend/src/pages/organization/SecretSharingPage/ShareSecretSection.tsx

```diff
@@ -0,0 +1,75 @@
+import { Helmet } from "react-helmet";
+import { useNavigate, useSearch } from "@tanstack/react-router";
+
+import { Badge, Tab, TabList, TabPanel, Tabs } from "@app/components/v2";
+import { ROUTE_PATHS } from "@app/const/routes";
+import { usePopUp } from "@app/hooks";
+
+import { RequestSecretTab } from "./components/RequestSecret/RequestSecretTab";
+import { ShareSecretTab } from "./components/ShareSecret/ShareSecretTab";
+
+enum SecretSharingPageTabs {
+  ShareSecret = "share-secret",
+  RequestSecret = "request-secret"
+}
+
+export const ShareSecretSection = () => {
+  const { popUp, handlePopUpToggle, handlePopUpClose, handlePopUpOpen } = usePopUp([
+    "createSharedSecret",
+    "deleteSharedSecretConfirmation",
+    "createSecretRequest",
+    "deleteSecretRequestConfirmation",
+    "revealSecretRequestValue"
+  ] as const);
```

Comment:
any reason why this is in the `ShareSecretSection` instead of in the respective tab components?